engine/schema/src/main/java/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -229,6 +231,8 @@ public DatabaseUpgradeChecker() {
                 .next("4.18.1.0", new Upgrade41810to41900())
                 .next("4.19.0.0", new Upgrade41900to41910())
                 .next("4.19.1.0", new Upgrade41910to41920())
+                .next("4.19.2.0", new Upgrade41920to41930())
+                .next("4.19.3.0", new Upgrade41930to41940())
```

Comment:
I am not sure this works, but we should avoid creating empty upgrades.
```suggestion
                .next("4.19.2.0", null)
                .next("4.19.3.0", new Upgrade41930to41940())
```